src/test/java/pixelitor/layers/TextLayerTest.java

```diff
@@ -177,4 +172,19 @@ public void commitSettings() {
 
         iconUpdates.check(0, 0);
     }
+
+    /**
+     * Newly added testCase
+     */
+    @Test
+    public void testHasRasterThumbnail() {
+        TextLayer textLayer = new TextLayer(comp);
+        boolean hasRasterThumbnail = textLayer.hasRasterThumbnail();
+        Assertions.assertFalse(hasRasterThumbnail);
+    }
+    @Test
+    public void testResize() {
+        TextLayer textLayer = new TextLayer(comp);
+        textLayer.resize(new Dimension(100, 100));
```

Comment:
If you check the implementation of this method, you can see, that it's not implemented, so this is a useless testcase.